tests/test-rest-attachments-controller.php

```diff
@@ -20,6 +20,17 @@ public function setUp() {
 		$this->contributor_id = $this->factory->user->create( array(
 			'role' => 'contributor',
 		) );
+		//matches core capabilities for subscriber role and adds upload_files
+		_x( 'File upload role', 'User role' );
```

Comment:
Correct. We'd either need to add a variable and pass it to the add_role function or just remove it since our goal here isn't really to test translations. I can modify it either way, but I'd lean towards removing it since it's not the goal of the test and won't really have an impact either way.